analytics.go

```diff
@@ -94,3 +151,45 @@ func (c *Client) getAggregatedData(ctx context.Context, analytics AnalyticsReque
 
 	return analyticsResponse, nil
 }
+
+// GetRawDataSingleIncident gets the raw analytics for the requested incident.
+func (c *Client) GetRawDataSingleIncident(ctx context.Context, id string) (RawData, error) {
```

Comment:
A few nitpicky comments about the function names: could we name these two functions after their [OpenAPI Operation Ids](https://github.com/PagerDuty/api-schema/blob/55f11455b56cd1a829e0a949ebba744087c75172/reference/REST/openapiv3.json#L18401): `getAnalyticsIncidentsById` and `getAnalyticsIncidents`?